crates/daphne-server/src/storage_proxy_connection/kv/cache.rs

```diff
@@ -30,12 +30,9 @@ pub struct Cache {
 }
 
 pub enum GetResult<T: 'static> {
-    NotFound {
-        /// Whether the caller should check KV for the value.
-        read_through: bool,
-    },
+    NotFound,
```

Comment:
Is this what you intend?
```suggestion
    /// The caller should read through to KV.
    NotFound,
```

Reply:
Don't think about it in terms of the cache telling the caller whether it should read through KV and more in terms of "there was a cache hit or a cache miss" and the caller then decides what it wants to do on either circumstance